src/include/storage/index/hash_index_builder.h

```diff
@@ -121,11 +122,13 @@ class HashIndexBuilder : public BaseHashIndex {
     std::unique_ptr<InMemFile> inMemOverflowFile;
     uint8_t slotCapacity;
     std::atomic<uint64_t> numEntries;
+    common::FileSystem* fs;
```

Comment:
We can remove `fs` from fields of the HashIndexBuilder. It should ideally be only passed to constructor to construct `fileHandle` and `inMemOverflowFile`.

Reply:
Removed fs from hashindexbuilder